### PR TITLE
fix: make "Learn about CPoS" link open in new tab

### LINKED ISSUE #971: Namadillo: "Learn about CPoS" link opens in same tab
The "Learn about CPoS" link in the Staking page opens in the same tab as Namadillo. This is not the same as the other external links in Namadillo, which open a new tab, and CPoS should probably do the same.
2024-08-07-111514_462x782_scrot

### PR DESCRIPTION
**Pull request by emccorson**
Fixes #971.

### CODE REVIEW

File: apps/namadillo/src/App/Sidebars/ValidatorDiversification.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const ValidatorDiversification = (): JSX.Element => {
         textHoverColor="black"
         backgroundHoverColor="cyan"
         href="https://namada.net"
```
Comment (euharrison) [inline apps/namadillo/src/App/Sidebars/ValidatorDiversification.tsx:21]: do you know if we have a better link for this? It looks like just a "lorem ipsum"

**Review by euharrison — APPROVED** [review]

File: apps/namadillo/src/App/Sidebars/ValidatorDiversification.tsx (hunk 1 above)
Comment (emccorson) [reply apps/namadillo/src/App/Sidebars/ValidatorDiversification.tsx:21]: As far as I know, we don't. I was thinking of reviewing all the links together and making a list for Awa or Chris to decide on, which is why I didn't touch the link in this PR. I'll open a new issue for it.

Comment (emccorson) [reply apps/namadillo/src/App/Sidebars/ValidatorDiversification.tsx:21]: https://github.com/anoma/namada-interface/issues/985

**Review by mateuszjasiuk — APPROVED** [review]

File: apps/namadillo/src/App/Sidebars/ValidatorDiversification.tsx (hunk 1 above)
Comment (euharrison) [reply apps/namadillo/src/App/Sidebars/ValidatorDiversification.tsx:21]: I found this now :) 
https://namada.net/blog/namada-cubic-proof-of-stake

**Pull request merged**